### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IvanObreshkov**

### CODE REVIEW

File: services/py-api/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## How to set up project:
+1. Run `poetry config virtualenvs.in-project true`, this tells poetry to create the virtual env in the project dir. You can learn more [here](https://python-poetry.org/docs/configuration/#virtualenvsin-project)
+2. If you have pyenv installed run `poetry config virtualenvs.prefer-active-python true`. You can learn more [here](https://python-poetry.org/docs/managing-environments/)
```
Comment (asynchroza) [inline services/py-api/README.md:3]: This is not part of the initialization script. I suggest you add it there.

**Review by Isak-Bego — COMMENTED** [review]
Is it really neccessary to know this info when you are running a Dev Container

**Comment by IvanObreshkov** [comment]
> Is it really neccessary to know this info when you are running a Dev Container

No, this is for those of us who run only the python API (only me I guess) 😔

**Comment by Isak-Bego** [comment]
> > Is it really neccessary to know this info when you are running a Dev Container
> 
> No, this is for those of us who run only the python API (only me I guess) 😔

Sucks to suck man

**Review by Isak-Bego — APPROVED** [review]
Nice enhancements to the documentation. The Dev Container builds sucessfully and runs as expected. I approve

**Pull request merged**